src/main/scala/nl/knaw/dans/api/sword2/Authentication.scala

```diff
@@ -25,7 +25,7 @@ object Authentication {
       throw new SwordError("http://purl.org/net/sword/error/MediationNotAllowed")
     }
     // temporary short-circuit
-    if (!(auth.getUsername == SwordProps.get("user")) || !(auth.getPassword == SwordProps.get("password"))) {
+    if (!(auth.getUsername == SwordProps("user")) || !(auth.getPassword == SwordProps("password"))) {
```

Comment:
A rather complicated boolean expression. How about

     auth.getUsername != SwordProps("user") || ...

?

Why a *temporary* short-circuit? Are you referring to the upcoming implementation using LDAP?